besluit/src/main/kotlin/com/ritense/besluit/web/rest/impl/BesluitResource.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2015-2022 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.besluit.web.rest.impl
+
+import com.ritense.besluit.web.rest.BesluitResource
+import com.ritense.besluit.web.rest.response.BesluitTypeResponse
+import com.ritense.openzaak.service.impl.ZaakTypeService
+import org.springframework.http.ResponseEntity
+
+class BesluitResource(
+    val zaakTypeService: ZaakTypeService,
+) : BesluitResource {
+
+    override fun getBesluitTypen(): ResponseEntity<List<BesluitTypeResponse>> {
+        val besluitTypes = zaakTypeService.getBesluitTypes().results
```

Comment:
This mapping should happen in the service itself

Reply:
I moved the mapping to a new service